### ISSUE TITLE
Unable to download magento through composer create-project command

### ISSUE THREAD

**Issue by emastyle** [ModestCoders/magento2-dockergento#16]
After successfully docker container installation following docs, trying to download/install magento through composer create-project, fails due to permission error while deleting vendor folder (see attachement). 
![dndm2txxsaa_v3m](https://user-images.githubusercontent.com/1888713/45613522-f5ea3e80-ba66-11e8-9454-436a68d8014f.jpg)

**Comment by jalogut** [comment]
Hi @emastyle 

We have developed a fix for that on [magento2-dockergento-console](
https://github.com/ModestCoders/magento2-dockergento-console/tree/develop). The fix is now on the develop branch and will be merged into master soon.

In the meantime, if want to check it out, you can do the following:

```
cd magento2-dockergento-console
git checkout develop && git pull
cd <your projects dir>
mkdir magento2-project
cd magento2-project
dockergento setup
dockergento start
dockergento create-project
```

If you can try it, please let us know if it worked.

**Comment by jalogut** [comment]
Hi @emastyle 

Issue is fixed in version `1.4.0` of `magento2-dockergento-console`. You can try again doing the following:

```
cd magento2-dockergento-console
git pull
cd <your projects dir>
mkdir magento2-project
cd magento2-project
dockergento setup
dockergento start
dockergento create-project
```

**Comment by emastyle** [comment]
Yes, it works 👍 Thank you very much.
